README.md

```diff
@@ -30,6 +29,12 @@ Jokes with animals. Submit your own, if they make me laugh, I'll merge them.
 
 ---
 
+**Q:** Why do fish never use the internet?
+
+**A:** Because they are afraid of the 'net'
+
+---
+
```

Comment:
This may not sound as funny as in persian. Please only keep the persian version.